scripts/build_lib_power.py

```diff
@@ -10,7 +10,7 @@
 import os.path
 
 PWR_NAMES = [
-    "VCC", "VDD", "AVCC", "1v2", "1v8", "3v3", "5v",  "12v", "VBATT", "VSHORE",
+    "VCC", "VDD", "AVCC", "1v2", "1v8", "3v3", "5v",  "12v", "VBATT", "VSHORE"
```

Comment:
Please could you restore the trailing comma so this file is unchanged by the pull request?